benches/blitzar_benchmarks.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023-present Space and Time Labs, Inc.
+// Copyright 2024-present Space and Time Labs, Inc.
```

Comment:
why is the date on the copyright being bumped?

Reply:
This is a result of a misunderstanding on my part. I see that [the date on the notice establishes how far back the claim is made](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated).

I am reverting this change now. Thank you.